Assets/Scripts/DataController.cs

```diff
@@ -93,6 +68,26 @@ public DateTime CurrentSimulationTime
     // For storing a copy of the last known time to limit updates
     private double lastTime;
 
+    // Scene-specific settings, set via SceneManagerComponent
+    private bool colorByConstellation = true;
+    private bool showConstellationConnections = true;
+    private int maxStars = 10000;
+    private float magnitudeScale = 0.5f;
+    private float magnitudeThreshold = 4.5f;
+    private float minMag;
+    private float maxMag;
+    private bool showHorizonView = false;
+    private float simulationTimeScale = 10f;
+    private float radius = 50;
+    public float Radius
+    {
+        get { return radius; }
+        private set { radius = value; }
+    }
+
+    private bool isReady = false;
```

Comment:
Were there issues tied to moving the sphere and scene loading?

Reply:
isReady was added mainly to stop fixed update from running until we've reset things